src/AppInstallerCLICore/ExecutionContext.h

```diff
@@ -139,7 +160,10 @@ namespace AppInstaller::CLI::Execution
     // arguments via Execution::Args.
     struct Context
     {
-        Context(std::ostream& out, std::istream& in) : Reporter(out, in) {}
+        Context(std::ostream& out, std::istream& in) : Reporter(out, in) { m_data[Data::ContextFlag] = ContextFlag::None; }
+
+        // Clone the reporter for this constructor.
+        Context(Execution::Reporter& reporter) : Reporter(reporter, Execution::Reporter::clone_t{}) {}

```

Comment:
Why are you making this public?  It wasn't to force the use of the Clone method.

Reply:
std::make_unique in the clone90 needs to access it 

---
In reply to: [503507085](https://github.com/microsoft/winget-cli/pull/588#discussion_r503507085) [](ancestors = 503507085)